beebop/app.py

```diff
@@ -1,9 +1,28 @@
-from flask import Flask, jsonify
+from importlib.metadata import files
+from flask import Flask, jsonify, request
+from flask_expects_json import expects_json
 from waitress import serve
+import json
+from redis import Redis
+import redis.exceptions as redis_exceptions
+from rq import Queue
+from rq.job import Job
+import os
 
 from beebop import versions
+from beebop import assignClusters
+from beebop.filestore import PoppunkFileStore
+import beebop.schemas
+schemas = beebop.schemas.Schema()
+
 
 app = Flask(__name__)
+redis = Redis()
+
+if os.environ.get('TESTING') == 'True':
```

Comment:
Is this a location that we write too? If so it's not 100% ideal as it means that each run of the test suite is not totally separate from the previous one. Can be dealt with later though